plonk/src/circuit/plonk_verifier/mod.rs

```diff
@@ -829,4 +832,24 @@ mod test {
 
         Ok(())
     }
+
+    // two circuit with the same statement should have the same extended permutation
+    // polynomials even with different variable assignment
+    // TODO(Chengyu): Copied from crate `jf-relation`, can we remove this
```

Comment:
Maybe we can set this API public in `jf-relation` so that we can call it in other crates? WDYT? @alxiong @zhenfeizhang

Reply:
I tried but failed before. Maybe because this function lies in `mod test`?

Reply:
I see, one approach is to move the API out of `mod test`, and use sth like this:
```
#[cfg(test)]
pub fn foo() { ... }

#[cfg(not(test))]
fn foo() { ... }
```
I'm not sure if this is the best practice though.

Reply:
Moved out. Thanks.

Reply:
yes, Binyi's solution is good for now. 
in the future, when we want to expose bunch of helper functions for testing, we can put them under:

```rust
#[cfg(test)]
mod test_utils;

// in test_utils
pub fn reuseable_test_helpers() {}
```

so that these functions can only be used in testing modules of other crates.